### PR TITLE
Add Zyxel cfg intf group get

### PR DESCRIPTION
**Pull request by elavaud**
##### ISSUE TYPE
 - New Template Pull Request

##### COMPONENT
Zyxel OS, cfg intf group get

### CODE REVIEW

File: ntc_templates/templates/zyxel_os_cfg_intf_group_get.textfsm [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Value INDEX (\d+)
+Value NAME (.+?)
+Value WAN_INTF (Any\sWAN|((WWAN|ADSL|VDSL),?)+)
+Value INTERFACE (.+(?<!\s))
```
Comment (itdependsnetworks) [inline ntc_templates/templates/zyxel_os_cfg_intf_group_get.textfsm:4]: Not sure what this is looking for, can you add an example to the tests?

File: ntc_templates/templates/zyxel_os_cfg_intf_group_get.textfsm [hunk 2]
```diff
@@ -0,0 +1,16 @@
+Value INDEX (\d+)
+Value NAME (.+?)
+Value WAN_INTF (Any\sWAN|((WWAN|ADSL|VDSL),?)+)
```
[the remark this answers is not in the document]
Comment (elavaud) [reply ntc_templates/templates/zyxel_os_cfg_intf_group_get.textfsm:3]: https://www.zyxel.com/us/en/uploads/images/VMG3925-B10A_UG_v5-11_ed1_2015-11-11.pdf

Chapter 15 starting page 183

File: ntc_templates/templates/zyxel_os_cfg_intf_group_get.textfsm (hunk 1 above)
Comment (elavaud) [reply ntc_templates/templates/zyxel_os_cfg_intf_group_get.textfsm]: Yip, I haven't spotted that one.
I corrected and added more examples to the tests

**Review by jmcgill298 — COMMENTED** [review]
needs rebase

**Comment by elavaud** [comment]
Grouped in #1142

**Pull request closed without merging**